src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.cs

```diff
@@ -24,7 +24,7 @@ internal sealed partial class WorkCoordinator
             private readonly Registration _registration;
             private readonly object _gate = new();
 
-            private readonly LogAggregator _logAggregator = new();
+            private readonly CountLogAggregator<WorkspaceChangeKind> _logAggregator = new();
```

Comment:
Yes, the WorkCoordinator has two different log aggregators! Until this, it was quite confusing what went in which...